apps/ledger-live-mobile/src/families/hedera/AssociateTokenFlow/01-SelectToken.tsx

```diff
@@ -37,7 +37,15 @@ const renderEmptyList = () => (
 export default function SelectToken({ navigation, route }: Props) {
   const { t } = useTranslation();
   const { account, parentAccount } = useSelector(accountScreenSelector(route));
-  const list = useMemo(() => listTokens().filter(t => t.parentCurrency.id === "hedera"), []);
+
+  const { data, loadNext } = useTokensData({
+    networkFamily: "hedera",
+    pageSize: 100,
+  });
+
+  const list = useMemo(() => {
+    return data?.tokens || [];
+  }, [data]);
```

Comment:
I think we could even only subscribe to the tokens change only
```suggestion
  }, [data?.tokens]);
```